scripts/dockers/qa/upgrade.py

```diff
@@ -0,0 +1,35 @@
+import time
+from JumpScale import j
+
+MANIFEST_URL = 'https://raw.githubusercontent.com/0-complexity/devmanifests/master/manifests/9.9.9.yml'
+
+scl = j.clients.osis.getNamespace('system')
+pcl = j.clients.portal.getByInstance('main')
+
+def wait_until_update_done(upgradeTime, timeout=180):
+    for _ in range(timeout):
```

Comment:
timeout is misleading variable name as its multiplied by your sleeptime it's more like retry loops